### ISSUE TITLE
Display sync errors more clearly?

### ISSUE THREAD

**Issue by michielbdejong** [NoelDeMartin/solid-focus#5]
Great app! And super fast to use! Are you using `SolidAuth.fetch` directly, without any of the libraries from https://github.com/inrupt/solid-lib-comparison#exploration-of-client-side-rdf-libraries?

It does scare me a little bit though, when I remove my cookies or switch off my wifi and then add another ToDo item to one of my lists, it looks like it was added successfully, even though that item was then not successfully stored on my pod (yet). Maybe it would make sense to show an error in this case, or at least a 'Syncing ...' message so a user knows their changes were not yet saved to their pod?

**Comment by NoelDeMartin** [comment]
Thanks for the nice words :). I am using soukai-solid, which is a library I implemented to interact with Solid Pods. I introduced it on the Solid forum [here](https://forum.solidproject.org/t/soukai-a-different-way-to-query-pods/2105), so you can read more about it there.

You are right on worrying about that, it's something I didn't tackle yet but I have in mind. I'll keep this issue open until I do something about it.

**Comment by michielbdejong** [comment]
Great, thanks!

**Comment by NoelDeMartin** [comment]
I've added the following functionality to the latest release:

![solid-focus-loading](https://user-images.githubusercontent.com/1517677/69913584-a872ab00-1439-11ea-930c-4f468cbcc203.gif)

I've been thinking what would be the best way to do this. For now I've decided to add some loading statuses when requests take longer than 1 second. Someone with a fast internet connection will have the same experience they had before, and someone with slow internet can rest easy with this visual feedback :).
